packages/ingest-mongodb-public/src/sources/snootySources.ts

```diff
@@ -148,6 +154,12 @@ export const snootyProjectConfig: LocallySpecifiedSnootyProjectConfig[] = [
     tags: ["docs", "mongodb-vscode", "vscode", "gui"],
     productName: "MongoDB for VS Code",
   },
+  {
+    type: "snooty",
+    name: "mcp-server",
+    tags: ["docs", "mongodb-mcp", "mcp"],
+    productName: "MongoDB MCP Server",
```

Comment:
@hschawe glad you did the strong typing on productName. otherwise, i would have forgotten to add the MCP server to the products list below.